lib/web_ui/test/canvaskit/h5vcc_test.dart

```diff
@@ -23,7 +24,7 @@ void testMain() {
     setUpAll(() async {
       // Set `window.h5vcc` to PatchedH5vcc which uses a downloaded CanvasKit.
       final CanvasKit downloadedCanvasKit = await downloadCanvasKit();
-      debugH5vccSetter = PatchedH5vcc(downloadedCanvasKit);
+      debugH5vccSetter = PatchedH5vcc.create(canvasKit: downloadedCanvasKit);
 
       // Monkey-patch the getH5vccSkSurface function of
       // `window.h5vcc.canvasKit`.
```

Comment:
This patching worries me. `canvasKit` and `getH5vccSkSurface` might be renamed by the compilers as their containers are Dart object. Using the `anonymous` class actually fixes the issue for `canvasKit`, but if `downloadedCanvasKit` is a Dart object, the issue still persists. Furthermore, if `getH5vccSkSurface` gets called anywhere in JS, it should be `allowInterop`'d.

Reply:
Okay, this should be cleaner now, ptal.

Reply:
LGTM!